src/column.h

```diff
@@ -62,6 +63,7 @@ class ColumnCharacter : public Column {
   ColumnCharacter(Rcpp::List opts_) : Column(Rcpp::CharacterVector()) {
     trim_ws = opts_["trim_ws"];
   }
+  virtual ~ColumnCharacter () = 0;
```

Comment:
I get build time error after this, since there's no subclass for ColumnCharacter, it needs a non-virtual destructor, right?

```
column.cpp:12:26: error: allocating an object of abstract class type 'ColumnCharacter'
    return ColumnPtr(new ColumnCharacter(var_opts));
                         ^
./column.h:66:11: note: unimplemented pure virtual method '~ColumnCharacter' in 'ColumnCharacter'
  virtual ~ColumnCharacter () = 0;
```